### ISSUE TITLE
Some background sequences start point is negative number

### ISSUE THREAD

**Issue by qxuchn** [vanheeringen-lab/gimmemotifs#76]
**Describe the bug**
When I chose XT9.1 (Xenopus_tropicalis_v9.1) as the genome, the genome random sequences always have some negative start sequences, for example: scaffold_2334 -99 100

/home/user/anaconda3/envs/regnetwork/lib/python3.6/site-packages/pyfaidx/\__init\__.py
```python
 617 def fetch(self, name, start, end):
 618         if self.read_ahead and not (name, start, end) in self:
 619             self.fill_buffer(name, start, end + self.read_ahead)
 620         print(name, start, end)                                                                                                                                                                               
 621         if (name, start, end) in self:
 622             seq = self.from_buffer(start, end)
 623         else:
 624             seq = self.from_file(name, start, end)
 625          
 626         return self.format_seq(seq, name, start, end)
 627
````
when I add line 620 in this function, it will print error line:
scaffold_2334 -99 100

**Error logs**
```bash 
 File "/home/user/anaconda3/envs/regnetwork/lib/python3.6/site-packages/gimmemotifs/scanner.py", line 569, in best_score
    for matches in self.scan(seqs, 1, scan_rc, normalize=normalize, gc=gc):
  File "/home/user/anaconda3/envs/regnetwork/lib/python3.6/site-packages/gimmemotifs/scanner.py", line 610, in scan
    self.set_meanstd(gc=gc)
  File "/home/user/anaconda3/envs/regnetwork/lib/python3.6/site-packages/gimmemotifs/scanner.py", line 354, in set_meanstd
    self.set_background(gc=gc)
  File "/home/user/anaconda3/envs/regnetwork/lib/python3.6/site-packages/gimmemotifs/scanner.py", line 451, in set_background
    fa = as_fasta(tmp.name, genome=genome)
  File "/home/user/anaconda3/envs/regnetwork/lib/python3.6/site-packages/gimmemotifs/utils.py", line 613, in as_fasta
    genome.track2fasta(seqs, tmpfa.name)
  File "/home/user/anaconda3/envs/regnetwork/lib/python3.6/site-packages/genomepy/functions.py", line 429, in track2fasta
    for seq in seqqer:
  File "/home/user/anaconda3/envs/regnetwork/lib/python3.6/site-packages/genomepy/functions.py", line 382, in _bed_to_seqs
    seq = self.get_spliced_seq(chrom, intervals, rc)
  File "/home/user/anaconda3/envs/regnetwork/lib/python3.6/site-packages/pyfaidx/__init__.py", line 1046, in get_spliced_seq
    chunks = [self.faidx.fetch(name, s, e) for s, e in intervals]
  File "/home/user/anaconda3/envs/regnetwork/lib/python3.6/site-packages/pyfaidx/__init__.py", line 1046, in <listcomp>
    chunks = [self.faidx.fetch(name, s, e) for s, e in intervals]
  File "/home/user/anaconda3/envs/regnetwork/lib/python3.6/site-packages/pyfaidx/__init__.py", line 624, in fetch
    seq = self.from_file(name, start, end)
  File "/home/user/anaconda3/envs/regnetwork/lib/python3.6/site-packages/pyfaidx/__init__.py", line 646, in from_file
    "Requested start coordinate must be greater than 1.")
pyfaidx.FetchError: Requested start coordinate must be greater than 1.
```

**Installation information (please complete the following information):**
 - OS: [Linux]
 - Installation [conda]
 - Version [develop]

**Additional context**

**Comment by simonvh** [comment]
Should be fixed in develop (db33455)

**Comment by qxuchn** [comment]
It is fixed. Thank you very much!
